### PR TITLE
SafeLoggable.getLogMessage is annotated as `@Safe`

### PR DESCRIPTION
**Pull request by carterkozak**
No difference in practice, just allows annotations to reinforce documentation/spec.
==COMMIT_MSG==
SafeLoggable.getLogMessage is annotated as `@Safe`
==COMMIT_MSG==

### CODE REVIEW

File: safe-logging/src/main/java/com/palantir/logsafe/SafeArg.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import javax.annotation.Nullable;
 
 /** A wrapper around an argument known to be safe for logging. */
+@Safe
 public final class SafeArg<T> extends Arg<T> {
```
Comment (carterkozak) [inline safe-logging/src/main/java/com/palantir/logsafe/SafeArg.java:23]: drive-by

File: safe-logging/src/main/java/com/palantir/logsafe/UnsafeArg.java [hunk 2]
```diff
@@ -19,6 +19,7 @@
 import javax.annotation.Nullable;
 
 /** A wrapper around an argument that is not safe for logging. */
+@Unsafe
 public final class UnsafeArg<T> extends Arg<T> {
```
Comment (carterkozak) [inline safe-logging/src/main/java/com/palantir/logsafe/UnsafeArg.java:23]: here as well

**Review by tpetracca — APPROVED** [review]
